lib/msal-node/src/client/ClientApplication.ts

```diff
@@ -101,12 +101,13 @@ export abstract class ClientApplication {
         };
         
         const authClientConfig = await this.buildOauthClientConfiguration(
-            validRequest.authority
+            validRequest.authority,
+            validRequest.correlationId
         );
-        this.logger.verbose("Auth client config generated");
         const authorizationCodeClient = new AuthorizationCodeClient(
             authClientConfig
         );
+        this.logger.verbose("Auth code client created", validRequest.correlationId);
```

Comment:
As there are not many logs in `msal-node` that use the flow client logger, I've decided to use the global logger for now with the updated correlation ID instead of instantiating a new `nodeRequestLogger` instance for each request. This can, of course, be changed in future PRs as the `Logger` changes.